README.md

```diff
@@ -2,9 +2,19 @@
 [![Apache 2.0 License](https://img.shields.io/badge/License-Apache_2.0-blue)](LICENSE)
 ![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/flock-community/wirespec/build.yml)
 
-# Wirespec
+<div align="center">
+<h1>
+  <br>
+  <a href="https://github.com/flock-community/wirespec"><img src="./images/wirespec-logo.svg" alt="Wirespec logo" width="200"/></a>
```

Comment:
```suggestion
  <a href="https://github.com/flock-community/wirespec"><img src="./images/wirespec-logo.svg" alt="" width="200"/></a>
```
alt information is not needed when it's purely visual and has no information that's relevant. 

https://www.w3.org/WAI/tutorials/images/decision-tree/